zombienet-tests/js_scripts/0015-claim_no_fee.js

```diff
@@ -0,0 +1,114 @@
+const { hexToU8a } = require('@polkadot/util');
+
+const { init_api, sudoInitClaim, claim, receivedEvents, submitExtrinsic, BlockUntil } = require('zkv-lib');
+const ReturnCode = {
+    Ok: 1,
+    ErrInitClaim: 2,
+    ErrClaim: 3,
+    ErrClaimedAmount: 4,
+};
+
+const MNEMONIC = 'poverty popular note inform state innocent grant crumble manage tornado primary list';
+
+async function run(nodeName, networkInfo, _args) {
+
+    const api = await init_api(zombie, nodeName, networkInfo);
+
+    // Build a keyring and import Alice's credential
+    let keyring = new zombie.Keyring({ type: 'sr25519' });
+    const alice = keyring.addFromUri('//Alice');
+
+    // Build some different kind of beneficiaries starting from 0 balance
+    const beneficiary_sr = keyring.addFromUri('//BeneficiarySR');
+    
+    const beneficiary_polkadot_js = keyring.addFromUri(MNEMONIC);
+
+    keyring = new zombie.Keyring({ type: 'ed25519' });
+    const beneficiary_ed = keyring.addFromUri('//BeneficiaryED')
+
+    keyring = new zombie.Keyring({ type: 'ecdsa' });
+    const beneficiary_ecdsa = keyring.addFromUri('//BeneficiaryEC')
+
+    // Build Beneficiaries Map
+    const beneficiaries_map = new Map();
+    beneficiaries_map.set(beneficiary_sr.address, '1000000000000000000');
+    beneficiaries_map.set(beneficiary_polkadot_js.address, '1000000000000000000');
+    beneficiaries_map.set(beneficiary_ed.address, '1000000000000000000');
+    beneficiaries_map.set(beneficiary_ecdsa.address, '1000000000000000000');
+
+    // Begin claim with beneficiary
+    const message = "I'm claiming my funds !";
+    let events = await sudoInitClaim(alice, beneficiaries_map, '10000000000000000000', message);
+    if (!receivedEvents(events)) {
+        console.log(`Failed to initialize claim`);
+        return ReturnCode.ErrInitClaim;
+    }
+
+    const keys = await api.query.claim.beneficiaries.keys();
+    const beneficiaries = keys.map(({ args: [beneficiaryId] }) => beneficiaryId);
+    console.log('all beneficiaries:', beneficiaries.join(', ')); 
+
+    // Claim SR
+    let signature = beneficiary_sr.sign(message);
+    events = await claim({Sr25519: beneficiary_sr.publicKey}, {Sr25519: signature});
+    if (!receivedEvents(events)) {
+        console.log(`Failed to claim`);
+        return ReturnCode.ErrClaim;
+    }
+
+    // Check beneficiary balance
+    let balance_beneficiary = (await api.query.system.account(beneficiary_sr.address))["data"]["free"];
+    if (balance_beneficiary != '1000000000000000000') {
+        console.log(`Beneficiary SR balance is ${balance_beneficiary}, expected 1000000000000000000`);
+        return ReturnCode.ErrClaimedAmount;
+    }
+
+    // Claim SR Prefixed
+    signature = "0xc89703e8763b08dfedd8e78959d1c1b28138628e58c7e4bc2c0eefd87d796c10e3fdb7bd35ef713d02c49959d3bfb2c559673a70e54156bc71bff5e7541f1e86";;
```

Comment:
It might be a good future reference to add a comment on how this signature was generated